flutter_modular/lib/src/core/interfaces/modular_route.dart

```diff
@@ -69,6 +69,28 @@ abstract class ModularRoute<T> {
   ///
   Map<String, String>? get params;
 
+  ///
+  /// Paramenter name: [queryParams]
+  ///
+  /// The parameters that can be transferred to another screen
+  ///
+  /// Type: Map<String, String>
+  ///
+  /// For more example http://example.com/help?id=12&rate=22
+  ///
+  Map<String, List<String>>? get queryParams;
```

Comment:
Why is the value of queryParams a list?

Reply:
It's to support urls such as` http://example.com?ids=23&ids=24&ids=25`. Notice that `ids` occurs multiple times so the values are put in a list. I think this is more appropriate to take care of all use cases. Using a string as the value means only the first value is used and other information is lost.